pkg/consensus/consensus.go

```diff
@@ -125,8 +127,8 @@ func NewService(cfg Config) (Service, error) {
 
 		transactions:     make(chan *transaction.Transaction, 100),
 		blockEvents:      make(chan *coreb.Block, 1),
-		network:          cfg.Chain.GetConfig().Magic,
-		stateRootEnabled: cfg.Chain.GetConfig().StateRootInHeader,
+		network:          cfg.ProtocolConfiguration.Magic,
+		stateRootEnabled: cfg.ProtocolConfiguration.StateRootInHeader,
```

Comment:
Do we need this (and previous) var if we have full ProtocolConfig in the context?

Reply:
Removed them.